data/step_function_inputs/mtbs-burn-severity.json

```diff
@@ -0,0 +1,42 @@
+[
+    {    "collection": "mtbs-burn-severity",
+        "discovery": "s3",
+        "prefix": "EIS/COG/Fire-Hydro/",
+        "bucket": "veda-data-store-staging",
+        "filename_regex": "mtbs_CONUS_2015.cog.tif",
+        "start_datetime": "2015-01-01T00:00:00Z",
+        "end_datetime": "2015-12-31T23:59:59Z",
+    },
+    {    "collection": "mtbs-burn-severity",
+        "discovery": "s3",
+        "prefix": "EIS/COG/Fire-Hydro/",
+        "bucket": "veda-data-store-staging",
+        "filename_regex": "mtbs_CONUS_2016.cog.tif",
+        "start_datetime": "2016-01-01T00:00:00Z",
+        "end_datetime": "2016-12-31T23:59:59Z",
+    },
+    {    "collection": "mtbs-burn-severity",
+        "discovery": "s3",
+        "prefix": "EIS/COG/Fire-Hydro/",
+        "bucket": "veda-data-store-staging",
+        "filename_regex": "2017.cog.tif",
```

Comment:
Looks to me like this cog name might be a bit off. Compare `mtbs_CONUS_2016.cog.tif` vs `2017.cog.tif`